crates/bin/pcli/src/command/ceremony.rs

```diff
@@ -58,6 +58,7 @@ async fn handle_bid(app: &mut App, to: Address, from: AddressIndex, bid: &str) -
     planner.output(value, to);
     let plan = planner
         .memo("E PLURIBUS UNUM".into())
+        .memo_return_address(app.config.full_viewing_key.payment_address(from).0)
```

Comment:
this is the fix :adhesive_bandage: 

everything else in this branch is tweaks to facilitate debugging, and confirming this _does_ in fact fix the issue.